services/workflows-service/scripts/workflows/full_kyb_process.ts

```diff
@@ -0,0 +1,310 @@
+      documents: [],
+    },
+    states: {
+      data_collection: {
+        tags: [StateTag.COLLECTION_FLOW],
+        on: {
+          start: [
+            {
+              target: 'run_cn_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.additionalInfo.country == `cn`', // @TODO: Work out companyNumber with Daniel
+                },
+              },
+            },
+            {
+              target: 'run_hk_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.additionalInfo.country == `hk`', // @TODO: Work out companyNumber with Daniel
+                },
+              },
+            },
+            {
+              target: 'auto_reject',
+            },
+          ],
+        },
+      },
+      run_cn_kyb: {
+        tags: [StateTag.PENDING_PROCESS],
+        on: {
+          // CONTINUE: [{ target: 'run_aml' }],
+          CONTINUE: [{ target: 'run_vendor_data_collection' }],
+          FAILED: [{ target: 'auto_reject' }],
+        },
+      },
+      run_hk_kyb: {
+        tags: [StateTag.PENDING_PROCESS],
+        on: {
+          // CONTINUE: [{ target: 'run_aml' }],
+          CONTINUE: [{ target: 'run_vendor_data_collection' }],
+          FAILED: [{ target: 'auto_reject' }],
+        },
+      },
+      run_vendor_data_collection: {
+        tags: [StateTag.PENDING_PROCESS],
+        on: {
+          CONTINUE: [{ target: 'manual_review' }],
+          FAILED: [{ target: 'auto_reject' }],
+        },
+      },
+      // run_aml: {
+      //   tags: [StateTag.PENDING_PROCESS],
+      //   on: {
+      //     CONTINUE: [{ target: 'run_ubos' }],
+      //     // @TODO: add 404 handling
+      //     FAILED: [{ target: 'auto_reject' }],
+      //   },
+      // },
+      // run_ubos: {
+      //   tags: [StateTag.PENDING_PROCESS],
+      //   on: {
+      //     CONTINUE: [{ target: 'run_kyb_enrichment' }],
+      //     FAILED: [{ target: 'auto_reject' }],
+      //   },
+      // },
+      manual_review: {
+        tags: [StateTag.MANUAL_REVIEW],
+        on: {
+          approve: 'approved',
+          reject: 'rejected',
+          revision: 'revision',
+        },
+      },
+      revision: {
+        tags: [StateTag.REVISION],
+        always: [
+          {
+            target: 'pending_resubmission',
+          },
+        ],
+      },
+      pending_resubmission: {
+        tags: [StateTag.REVISION],
+        on: {
+          RESUBMITTED: 'manual_review',
+        },
+      },
+      approved: {
+        tags: [StateTag.APPROVED],
+        type: 'final' as const,
+      },
+      rejected: {
+        tags: [StateTag.REJECTED],
+        type: 'final' as const,
+      },
+      auto_reject: {
+        tags: [StateTag.REJECTED],
+        type: 'final' as const,
+      },
+    },
+  },
+  extensions: {
+    apiPlugins: [
+      {
+        name: 'asia_verify_cn_kyb',
```

Comment:
Talk Omri about how its saved in the context, lets refactor it a bit if possible